webservices/docs.py

```diff
@@ -1218,11 +1218,15 @@
 '''
 
 COMMUNICATION_COST = '''
-52 U.S.C. 30118 allows "communications by a corporation to its stockholders and
-executive or administrative personnel and their families or by a labor organization
-to its members and their families on any subject," including the express advocacy of
-the election or defeat of any Federal candidate.  The costs of such communications
-must be reported to the Federal Election Commission under certain circumstances.
+<b>Deprecation notice: The pagination schema used for this endpoint will change to be in line with \
```

Comment:
Are we changing the pagination schema for this endpoint or deprecating it in favor of `communication_costs` per #4203?

Reply:
We will deprecate it in favor of `/communication_cost/` which will implement the new pagination scheme. Once that new endpoint is in place, we will update API users by email and swagger docs per completion criteria in #4203

Reply:
Should we tell users that we will "deprecate it in favor of `/communication_cost/` which will implement the new pagination scheme" rather than "The pagination schema used for this endpoint will change"?

Reply:
Hi @lbeaufort this is _precisely_  what we'll update  the deprecation notice to as part of the  completion criteria of #4203 **after** that endpoint is made available. Until then, we don't want to refer to a non-existent endpoint. The users will be given the advance notice per our deprecation policy.

Reply:
We are not changing the pagination schema for this endpoint. We are replacing it for a new endpoint with a different pagination schema. I agree with @lbeaufort that the current language is unclear about what users should expect when the change is eventually made. We should not be specific at all beyond "deprecation notice" if we can't tell them at this time that the endpoint will be replaced. 
cc: @patphongs